### DISCUSSION TITLE
i18n

### DISCUSSION THREAD

**Discussion by avm99963** [joeywatts/disable-autogain-control-extension#6] [Ideas]
Hi!

Awesome extension, thanks for developing it!

I was thinking of translating the extension to Spanish and Catalan (my native languages), so would it be fine if I made a PR to implement i18n via the [chrome.i18n](https://developer.chrome.com/docs/extensions/reference/i18n/) API and then make another one with the translations?

Cheers :-)

**Comment by joeywatts** [comment]
Sure thing, that would be appreciated!
